conda.yml

```diff
@@ -5,7 +5,7 @@ channels:
   - https://conda.anaconda.org/pyg
   - https://conda.anaconda.org/conda-forge
 dependencies:
-  - python>=3.7,<3.8
+  - python>=3.7,<3.9
```

Comment:
good idea, we can consider testing with 3.9 for a later PR and eventually relax the bound